### ISSUE TITLE
Outstanding checklist after #168

### ISSUE THREAD

**Issue by smishr** [xKDR/Survey.jl#173]
There are still a few problems left:
- Precompiling the package gives a warning because `quantile` is defined twice. That is mentioned in issue #171 and will be fixed by PR #172.
- The documentation is far from complete, both the docstrings and the files inside `docs/src`. We should do a separate PR on documentation.
- The README is outdated. @ayushpatnaikgit I can do it, but I suggest doing it in a separate PR.
- The `show` methods for `SurveyDesign` and `ReplicateDesign` could be greatly improved. This will mean changing the doctests again.
